src/store/sagas/upgrade.js

```diff
@@ -25,8 +24,9 @@ function* upgradeSuccess(action: ExtractReturn<typeof upgrade.upgradeSuccess>):
 }
 
 function* submitMnemonicRequest(
-  action: ExtractReturn<typeof upgrade.submitMnemonicRequest>,
+// action: ExtractReturn<typeof upgrade.submitMnemonicRequest>,
```

Comment:
same, about removing lines

Reply:
lines will be uncommented in few days, once we are ready to update these screens